Account/Status.ts

```diff
@@ -0,0 +1,22 @@
+import { isly } from "isly"
+
+export type Status = {
+	mode: "active" | "frozen" | "closed"
```

Comment:
Use the `Mode` and a `Reason` type. Reason can be defined as 
```
export type Reason = "overdraft" | "other" | (string & Record<never, never>)
```
to fool typescript into giving type suggestions

Reply:
Check!